test/test_model.py

```diff
@@ -118,11 +137,6 @@ def test_unit_relation_data(self):
         ])
 
     def test_remote_app_relation_data(self):
-        self.backend = ops.model.ModelBackend()
-        meta = ops.charm.CharmMeta()
-        meta.relations = {'db0': None, 'db1': None, 'db2': None}
-        self.model = ops.model.Model('myapp/0', meta, self.backend)
```

Comment:
Why are these completely irrelevant, and copy & pasted so much? Error?

Reply:
We used to have some tests that used fake tools with a real model backend instance and some that used [FakeModelBackend](https://github.com/canonical/operator/pull/61/files#diff-2bdcc3da19a2e638a126aab53c420862L20). Because of that, some test cases had to replace `self.backend` initialized to `FakeModelBackend` with an instance of `ModelBackend`. The point of #61 was to make all test cases use fake hook tools which it did but some test cases still had the `ModelBackend` initialization logic in-place which is no longer relevant.

In short: this particular change is here to clean up old test cases and move the common initialization code to `setUp`.